### PR TITLE
feat: use nginx as upstream in the example of  docker-compose

### PR DESCRIPTION
**Pull request by gxthrj**
- [x] Use nginx as upstream in the example of  docker-compose .
- [x] Affirm that example is only used for functional verification .

### CODE REVIEW

File: example/docker-compose-alpine.yml [hunk 1]
```diff
@@ -39,21 +39,27 @@ services:
         ipv4_address: 172.18.5.10
 
   web1:
-    image: ruby:2-alpine
-    command: sh -c "mkdir -p /tmp/www && echo 'web1' > /tmp/www/web1.txt && ruby -run -ehttpd /tmp/www -p8000"
+    image: fabiocicerchia/nginx-lua
```
Comment (moonming) [inline example/docker-compose-alpine.yml:42]: why not use nginx image？

**Review by moonming — CHANGES_REQUESTED** [review]

File: example/docker-compose-alpine.yml (hunk 1 above)
Comment (gxthrj) [reply example/docker-compose-alpine.yml:42]: Done. change to nginx offical image.

File: example/upstream/web2.conf [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+        location / {
+            return 200 "hello web2";
+        }
+
+        location /static/ {
+            alias static/;
+        }
+    }
+}
```
Comment (tokers) [inline example/upstream/web2.conf:24]: Should add the line break.

Comment (gxthrj) [reply example/upstream/web2.conf]: Done.

**Review by moonming — APPROVED** [review]

**Review by tokers — APPROVED** [review]

**Pull request merged**